apps/backend/src/websocket/server.py

```
@@ -0,0 +1,246 @@
+"""
+WebSocket Server
+
+Main WebSocket server implementation for real-time infrastructure monitoring.
+Provides WebSocket endpoints for client connections and message handling.
+"""
+
+import asyncio
+import logging
+import json
+from typing import Any, Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)